docs/values-files/values-microk8s.yaml

```diff
@@ -0,0 +1 @@
+../../codacy/values-microk8s.yaml
```

Comment:
git symbolic link shenanigans

Reply:
Git just stores the destination of the symbolic link. It doesn't duplicate the file.